source/windows/file.c

```diff
@@ -13,7 +13,19 @@
 #include <stdio.h>
 #include <windows.h>
 
+static bool s_is_string_non_empty(const struct aws_string *str) {
+    return str != NULL && str->len > 0;
+}
+
+static bool s_is_wstring_non_empty(const struct aws_wstring *str) {
```

Comment:
ooc, this looks to be identical to the above s_is_string_non_empty(). Is there a reason to have a separate function for aws_wstring? Or can we use the above for both without issue?

Reply:
Ah, gotcha. That makes sense =P